src/wallet/sqlite.h

```diff
@@ -166,6 +166,15 @@ class SQLiteDatabase : public WalletDatabase
     void IncrementUpdateCounter() override { ++nUpdateCounter; }
 
     std::string Filename() override { return m_file_path; }
+    /** Return paths to all database created files */
+    std::vector<fs::path> Files() override
+    {
+        std::vector<fs::path> files;
+        files.emplace_back(m_dir_path / fs::PathFromString(m_file_path));
+        files.emplace_back(m_dir_path / "database");
+        files.emplace_back(m_dir_path / "db.log");
```

Comment:
In 788bb48f6712c39995831020dc36297f88c42f71 "wallet: introduce method to return all db created files"

These are created only by BDB.

SQLite makes only `wallet.dat` and `wallet.dat-journal`.